update-charts-dep.sh

```diff
@@ -41,6 +41,10 @@ echo "Updating Account Lookup Service..."
 helm dep up ./account-lookup-service
 checkCommandResult
 
+echo "Updating ALS Oracle Pathfinder..."
```

Comment:
You need to update the publish.sh script with a similar update.

Reply:
added to package.sh